exercise02/arr_manipulation.py

```diff
@@ -0,0 +1,45 @@
+
+class Arr_manipulation:
+
+    def __init__(self, arr) -> None:
+        self.arr = arr
+
+    def element_at(self, index):
+        try:
+            if index < 0 or index > len(self.arr):
+                return "invalid input"
+            return self.arr[index]
+        except Exception:
+            return "error"
```

Comment:
Raise exceptions instead of returning a string value, also you could use a defined exception class that helps in understanding what error was thrown